test/parallel/test-typescript.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const { spawnPromisified } = require('../common');
+const fixtures = require('../common/fixtures');
+const { strictEqual } = require('node:assert');
+const { test } = require('node:test');
+
+test('execute a typescript file', async () => {
+  const {
+    code,
+    stderr,
+    stdout,
+  } = await spawnPromisified(process.execPath, [
+    '--experimental-typescript',
+    fixtures.path('typescript/test-typescript.ts'),
+  ], { stdio: ['inherit'] });
```

Comment:
Since we don't use `stdin`, no need for that.
```suggestion
  ]);
```